contracts/contracts/PlasmaMVP.sol

```diff
@@ -69,6 +70,7 @@ contract PlasmaMVP {
         uint256 amount;
         uint256 committedFee;
         uint256 createdAt;
+        uint256 ethBlockNum;
```

Comment:
This should be a pr added to the rootchain as well

Reply:
Will do after this gets merged